oak-it/src/test/java/org/apache/jackrabbit/oak/composite/CompositeNodeStoreTest.java

```diff
@@ -18,9 +18,9 @@
  */
 package org.apache.jackrabbit.oak.composite;
 
-import static com.google.common.base.Predicates.compose;
-import static com.google.common.collect.Iterables.filter;
 import static com.google.common.collect.Lists.newArrayList;
+import static org.apache.jackrabbit.guava.common.base.Predicates.compose;
+import static org.apache.jackrabbit.guava.common.collect.Iterables.filter;
```

Comment:
for the sake of easy review it would be preferable if the imports would remain on the same line (even if that makes the order a bit off.

Reply:
Yes. Will improve.